pkg/monitor/payload/monitor_payload.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package payload
+
+import (
+	"bytes"
+	"encoding/gob"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	// EventSample is equivalent to PERF_RECORD_SAMPLE
+	EventSample = 9
+	// RecordLost is equivalent to PERF_RECORD_LOST
+	RecordLost = 2
+)
+
+// Meta is used by readers to get information about the payload.
+type Meta struct {
+	Size uint32
+	_    [28]byte // Reserved 28 bytes for future fields.
```

Comment:
I was asking @tgraf for feedback on Slack and he wanted to use 32 bytes instead of just a 4 byte length. I don't know what those 28 remaining bytes will be used for in the future.